_data/Ankit-Kharb/Ankit-Kharb.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Ankit Kharb
+institution: J.C Bose University of science and technology,faridabad,haryana,india
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **69** characters.